extensions/optaplanner-jsonb/runtime/src/main/java/io/quarkus/optaplanner/jsonb/OptaPlannerJsonbConfigCustomizer.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.optaplanner.jsonb;
+
+import javax.inject.Singleton;
+import javax.json.bind.JsonbConfig;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.persistence.jsonb.api.score.buildin.bendable.BendableScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.bendablebigdecimal.BendableBigDecimalScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.bendablelong.BendableLongScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardmediumsoft.HardMediumSoftScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardmediumsoftbigdecimal.HardMediumSoftBigDecimalScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardmediumsoftlong.HardMediumSoftLongScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardsoft.HardSoftScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardsoftbigdecimal.HardSoftBigDecimalScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardsoftdouble.HardSoftDoubleScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.hardsoftlong.HardSoftLongScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.simple.SimpleScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.simplebigdecimal.SimpleBigDecimalScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.simpledouble.SimpleDoubleScoreJsonbAdapter;
+import org.optaplanner.persistence.jsonb.api.score.buildin.simplelong.SimpleLongScoreJsonbAdapter;
+
+import io.quarkus.jsonb.JsonbConfigCustomizer;
+
+/**
+ * OptaPlanner doesn't use JSON-B, but it does have optional JSON-B support for {@link Score}, etc.
+ */
+@Singleton
+public class OptaPlannerJsonbConfigCustomizer implements JsonbConfigCustomizer {
+
+    @Override
+    public void customize(JsonbConfig config) {
+        // TODO use getScoreAdaptorList() once released to avoid hardcoding the Score implementations
+        config.withAdapters(new BendableScoreJsonbAdapter(),
```

Comment:
@cuijulian (very soft suggestion - as we need to wait for optaplanner 7.35 to go out anyway): would you mind adding a hint in this TODO when it will be possible to fix it? For example: "when upgrading to optaplanner 7.36" (that TODO fix didn't make 7.35 which is being released). Before it was merged, I'd use the Jira url.
Also it's unclear which class has "getScoreAdaptorList()", making it hard for others your yourself in 6 months to figure out this TODO easily.

Reply:
Good point. I may as well also bump the `optaplanner-bom` version to 7.35 in this PR before it get merged.